prov/gni/src/gnix_freelist.c

```diff
@@ -71,7 +71,7 @@ int __gnix_fl_refill(struct gnix_freelist *fl, int n)
 	 */
 	elems = calloc((n+1), fl->elem_size);
 	if (elems == NULL) {
-		ret = -FI_ENOMEM;
+	b	ret = -FI_ENOMEM;
```

Comment:
@sungeunchoi The bot didn't build this commit. I'm still getting things setup here and didn't build the most recent commit before pushing.